conf.py

```diff
@@ -16,11 +14,10 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Common Package Specification'
-copyright = u'2023, Matthew Woehlke'
+project = 'Common Package Specification'
+copyright = '2023, Matthew Woehlke'
 
-release = '.'.join(map(str, [0, 9, 0]))
-version = re.match('\\d+\.\\d+', release).group(0)
+release = version = "0.9.0"
```

Comment:
```suggestion
release = version = '.'.join(map(str, [0, 9, 0]))
```

(Also, we're using `'` for strings everywhere else... but I'd like to keep the int-tuple, please.)

Reply:
I used `"` because if you'd ever like to use a formatter, `black` is just the way to go.

Reply:
Sorry for the delayed response to this. I've used what's currently in mainstream, however flake8's developers have become increasingly hostile to the python developer community over the years, refusing to even support the pyproject.toml file and claiming that `pip` needs to change it's entire behavior before they'll even consider supporting it. `ruff`, `black`, and others are more preferable, and if anyone from the python space ever ends up here you'll end up with nagging regarding these.